src/renderer/screens/manager/AppsList/AppsList.js

```diff
@@ -83,6 +85,10 @@ const AppsList = ({
   const { push } = useHistory();
   const { search } = useLocation();
   const reduxDispatch = useDispatch();
+  const currenciesAccountsSetup = useSelector(currenciesSelector);
+  const disableInstallBanner = every(state.installed, app =>
+    some(currenciesAccountsSetup, currency => currency.id === state.appByName[app.name].currencyId),
```

Comment:
should we disable if you already added one account from the installed list? :thinking: